src/style.css

```
@@ -1 +1,14 @@
 /* styles go here */
+
+.form__group .form__field:not(:last-child) {
```

Increase specificity is useless :)
 
```suggestion
.form__field:not(:last-child) {
```